Client/game_sa/CGameSA.h

```diff
@@ -333,7 +333,7 @@ class CGameSA : public CGame
     CExplosionManager*              m_pExplosionManager;
     C3DMarkers*                     m_p3DMarkers;
     CRenderWareSA*                  m_pRenderWare;
-    CHandlingManager*               m_pHandlingManager;
+    std::shared_ptr<CHandlingManager> m_HandlingManager;
```

Comment:
can be unique_ptr with access by referense

Reply:
Should I change this on the server?

Reply:
> Should I change this on the server?

@TheNormalnij

Reply:
yes